frontend/src/views/Pages/AccountOverview.vue

```diff
@@ -41,7 +41,6 @@
   </div>
 </template>
 <script>
-import GddStatus from './AccountOverview/GddStatus.vue'
```

Comment:
Why?

Reply:
ich habe es in der nav menu leiste im mobielen und in der sidebar in der grossen ansicht. 
da dachte ich mir das kann erstmal weg ..